templates/swift/Example/PROJECT/AppDelegate.swift

```diff
@@ -10,37 +10,37 @@ import UIKit
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
-
+    
     var window: UIWindow?
```

Comment:
I'm not _too_ bothered, but it might be worth just removing the changes from this file entirely